web/organizations.go

```diff
@@ -30,7 +30,6 @@ func (s *AutograderService) getAvailableOrganizations(ctx context.Context, sc sc
 		course, err := s.db.GetCourseByOrganizationID(org.ID)
 		if err != nil {
 			s.logger.Errorf("couldn't fetch course from database: %v", err)
-			continue
```

Comment:
Sorry about this. It is probably because the error from the database is `gorm.ErrRecordNotFound`, which is what we want. So we could instead check for this error type and only continue if `err` is different from record not found. (We should only log an error if a different error message than record not found is discovered.)